src/Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorApp.tsx

```diff
@@ -26,6 +28,30 @@ export const ArtworkErrorApp: React.FC<ArtworkErrorAppProps> = ({
 
   const artworkSlug = artwork?.slug
 
+  // Artwork paths are excluded from regular pageview tracking via
+  // `trackingMiddleware`, since `ArtworkApp` does some custom tracking.
+  // However, we still want to track pageviews for error pages.
+  // This is mostly cribbed from `ArtworkApp`.
+  const trackPageview = useCallback(() => {
+    const path = window.location.pathname
+
+    if (typeof window.analytics !== "undefined") {
+      const properties: any = {
+        path,
```

Comment:
I went with the most basic of properties here, I can double-check with @abhitip if this is sufficient for these.